pom.xml

```diff
@@ -51,8 +51,8 @@
         <main.user.dir>${user.dir}</main.user.dir>
         <maven.staging.plugin>1.6.7</maven.staging.plugin>
         <maven.gpg.pluign>1.5</maven.gpg.pluign>
-        <coveralls.maven.plugin>3.1.0</coveralls.maven.plugin>
-        <cobertura.maven.plugin>2.6</cobertura.maven.plugin>
+        <coveralls.maven.plugin>3.2.1</coveralls.maven.plugin>
+        <cobertura.maven.plugin>2.7</cobertura.maven.plugin>
```

Comment:
It seems that we are using jacoco to generate the coverage report, do we still need `coveralls` and `cobertura` plugin?

![image](https://user-images.githubusercontent.com/584455/108855119-9626f300-7623-11eb-893c-3108a84bf34f.png)

Reply:
it could remove those two plugins.